.github/workflows/image-builder.yaml

```diff
@@ -0,0 +1,38 @@
+name: image-builder
+
+on:
+  schedule:
+    - cron: "0 6 */14 * *"
+jobs:
+  image-builder:
+    runs-on: [self-hosted, linux]
+    steps:
+    - name: Checkout image-builder
+      uses: actions/checkout@v3
+      with:
+        repository: 'outscale-vbr/image-builder'
+        path: "add-outscale-image"
+        ref: add-outscale-image
+    - name: Use latest Packer
+      uses: hashicorp-contrib/setup-packer@v1
+      with:
+        packer-version: 1.7.10
```

Comment:
Ansible and goss are installed through the makefile, why not packer too ?

Reply:
Anisble and goss are install in image-builder.
https://github.com/outscale-vbr/image-builder/blob/2c1b53bb06823d33609cfa9f700d1babe4dca9a9/images/capi/Makefile#L79

Reply:
When this PR is merged , ok . (https://github.com/outscale-dev/cluster-api-provider-outscale/pull/55)

Reply:
done